targets/posix/nanoCLR/Target_HAL.cpp

```
@@ -0,0 +1,24 @@
+//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include <iostream>
+#include <string>
+
+void HAL_Windows_Debug_Print(const char *szText)
```

Cppcheck (2.20.0)

[style] The function 'HAL_Windows_Debug_Print' is never used.

(unusedFunction)